.github/workflows/01-hello-world.yml

```diff
@@ -0,0 +1,77 @@
+# With thanks to https://gist.github.com/weibeld/f136048d0a82aacc063f42e684e3c494
+name: build-and-release
```

Comment:
I should rename the file name of his workflow as well. `01-hello-world.yml` was of course the name of the on-line example that I used as a starting template.

Reply:
Done.